frontend/src/metabase/admin/datamodel/selectors.js

```diff
@@ -3,17 +3,17 @@ import { createSelector } from 'reselect';
 import { computeMetadataStrength } from "metabase/lib/schema_metadata";
 
 
-const segmentsSelector         = state => state.datamodel.segments;
-const metricsSelector          = state => state.datamodel.metrics;
+const segmentsSelector         = (state, props) => state.datamodel.segments;
```

Comment:
Hmm... adding props to every selector whether or not it's using props just for the sake of consistency seems a bit excessive. Is there some other reason for doing this that I'm missing? 

Don't feel too strongly either way though, so feel free to keep it this way if you disagree. Changing everything back might be a bit of a pain.